README.md

```diff
@@ -2,7 +2,7 @@
 
 Rock Paper Scissors Lizard Spock using MPC.
 
-https://voltrevo.github.io/mpc-lizard-spock
+Try it: https://mpc.pse.dev/apps/lizard-spock
 
-Built using [MPC framework](https://github.com/voltrevo/mpc-framework) using a
+Built using [MPC framework](github.com/privacy-scaling-explorations/mpc-framework) using a
```

Comment:
```suggestion
Built using [MPC framework](https://github.com/privacy-scaling-explorations/mpc-framework) using a
```

prefix is required, otherwise this gets interpreted as a relative link (current_location/github.com/etc)

btw, are you in the pse org? if so you can make a branch against the original repo instead of forking, then I can commit simple fixes like this directly to your branch before merging
(this distinction can be a bit unintuitive, lmk if I should clarify)

Reply:
nvm, looks like I can commit to your fork directly haha :)